src/shared/clients/codecatalystClient.ts

```diff
@@ -38,24 +38,33 @@ interface CodeCatalystConfig {
     readonly gitHostname: string
 }
 
-export function getCodeCatalystConfig(): CodeCatalystConfig {
-    const stage = DevSettings.instance.get('cawsStage', 'prod')
-
-    if (stage === 'gamma') {
-        return {
-            region: 'us-west-2',
-            endpoint: 'https://public.codecatalyst-gamma.global.api.aws',
-            hostname: 'integ.stage.REMOVED.codes',
-            gitHostname: 'git.gamma.source.caws.REMOVED',
-        }
-    } else {
-        return {
-            region: 'us-east-1',
-            endpoint: 'https://public.codecatalyst.global.api.aws',
-            hostname: 'codecatalyst.aws',
-            gitHostname: 'codecatalyst.aws',
-        }
+export function getCodeCatalystConfig(): Readonly<CodeCatalystConfig> {
+    return getCodeCatalystConfigFromSettings(DevSettings.instance)
+}
+
+export function getCodeCatalystConfigFromSettings(settings: DevSettings): Readonly<CodeCatalystConfig> {
+    const defaultConfig = {
+        region: 'us-east-1',
+        endpoint: 'https://public.codecatalyst.global.api.aws',
+        hostname: 'codecatalyst.aws',
+        gitHostname: 'codecatalyst.aws',
     }
+    const devSetting = 'codecatalystService'
```

Comment:
We may want to export this name so it can be used in tests and maintain a single source

Reply:
Will update.

Reply:
actually devsettings are "strongly typed" here: https://github.com/aws/aws-toolkit-vscode/blob/a2e3936e84a6c34bc73b0f94c45bdf31700dec13/src/shared/settings.ts#L642-L653